### DISCUSSION TITLE
issues with example-project-rollup

### DISCUSSION THREAD

**Discussion by leblancmeneses** [adobe/spectrum-web-components#2341] [Q&A] [answered]
After launching [example-project-rollup](https://github.com/adobe/spectrum-web-components/tree/main/projects/example-project-rollup)

```
npm run start
```

The drop downs error with:
```
floating-ui.core.esm.js:97 Uncaught (in promise) ReferenceError: process is not defined
    at computePosition (floating-ui.core.esm.js:97:3)
    at async ActiveOverlay.updateOverlayPosition (ActiveOverlay.ts:472:53)
```

is there a proper solution to this?  other than adding ```<script>var process = {env: {}};</script>```   https://github.com/floating-ui/floating-ui/issues/933

Also ```start:build```  fails with:

```
> example-project-rollup@0.5.14 postbuild
> mv dist/*.css dist/styles.css

mv: rename dist/*.css to dist/styles.css: No such file or directory
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: example-project-rollup@0.5.14 
npm ERR!   at location: /Users/leblancmeneses/repos/spectrum-web-components/projects/example-project-rollup 
```

Thanks

**Comment by Westbrook** [comment, marked as answer]
Roughly like so: https://github.com/adobe/spectrum-web-components/pull/2342/commits/942415cbce42710ed507ffcdee274e534a4d4b9e.

This PR should land a little later today and you'll be good to go! Thanks for reaching out, and don't forget to share what you've been making with Spectrum with the rest of the community. 🙇🏼

**Reply by leblancmeneses** [reply]
Thanks for fixing the example project.  I ended up using what was also documented in the README ```npm init @open-wc```. The command generated a usable template.  However, now that you got the fix I might move back because I want to externalize the custom svg icons. 

Here is what I built yesterday with this awesome library!  "app-launcher"
<img width="1080" alt="Screen Shot 2022-06-07 at 10 07 15 AM" src="https://user-images.githubusercontent.com/730804/172415397-cacdf7ca-bcd3-4379-bca4-fd7af38b84d9.png">

Thanks again!

**Reply by Westbrook** [reply]
Nice work! Two quick notes as you make on your way...

1. It looks like you're not using the `slot="icon"` attribute in your `<sp-button>` this will help clean up the visual delivery a bit.
2. If looks like you're leveraging `<sp-icon>` directly. You can leverage the elements for each icon in a way that makes them really easy to reduce you page build. Check out https://opensource.adobe.com/spectrum-web-components/components/icons-workflow/#find-an-icon for more info on that.

Best of luck with your work!
